torch/fx/experimental/param_fetch.py

```diff
@@ -0,0 +1,42 @@
+from torch.fx.graph_module import GraphModule
+from typing import Any, Dict, List, Tuple
+import torch.nn as nn
+
+
+"""This dict maps the nn.Module class name to the attribute name list that we want to fetch for lowering.
+The first integer in the tuple is the version number of the nn.Module class when we create the parameter list.
+If there's a version mismatch then it means the parameter names in the book might be mismatched with nn.Module.
+"""
+module_fetch_book: Dict[str, Tuple[int, List[str]]] = {
+    "Linear": (1, ["weight", "bias"]),
+    "Conv2d": (1, ["weight", "bias", "kernel_size", "stride", "padding", "dilation", "groups", "padding_mode"]),
+    "BatchNorm2d": (2, ["weight", "bias", "running_mean", "running_var"]),
+}
+
+def fetch_module(mod: nn.Module) -> Dict[str, Any]:
+    """If `mod` is in `module_fetch_book`, fetch the mod's attributes that in the `module_fetch_book`
+    after checking module's version is compatible with the `module_fetch_book`.
+    """
+    parameters: Dict[str, Any] = {}
+    parameters["name"] = type(mod).__name__
+
+    if type(mod).__name__ in module_fetch_book:
+        _version, param_to_fetch = module_fetch_book[type(mod).__name__]
+        if _version != mod._version:
+            print(f"Fetcher version {_version} try to fetch {type(mod).__name__} version {mod._version}")
+        for attr in param_to_fetch:
+            parameters[attr] = getattr(mod, attr)
+
+    return parameters
+
+def fetch(fx_module: GraphModule):
```

Comment:
Similarly, this could be named better. Something like `lift_lowering_parameters_to_nodes`